docs/output.md

```diff
@@ -342,6 +342,120 @@ The saved files are the _good_ files, passing the `dust` or `entropy` filter tre
+  - `<sample_id>.classifiedreads.txt`: A list of read IDs and the hits each read had against each database for a given sample
+
+</details>
+
+The main taxonomic classification file from Kraken2 is the `_combined_reports.txt` or `*report.txt` file. The former provides you the broadest over view of the taxonomic classification results across all samples against a single database, where you get two columns for each sample e.g. `2_all` and `2_lvl`, as well as a summarised column summing up across all samples `tot_all` and `tot_lvl`. The latter gives you the most information for a single sample. The report file is also used for the taxpasta step.
+
+You will only receive the `.fastq` and `*classifiedreads.txt` file if you supply `--metagenomics_kraken_savereads` and/or `--metagenomics_kraken_save_readclassifications` parameters to the pipeline.
+
+#### KrakenUniq
+
+[KrakenUniq](https://github.com/fbreitwieser/krakenuniq) (formerly KrakenHLL) is an extenson to the fast k-mer-based classification [Kraken](https://github.com/DerrickWood/kraken) with an efficient algorithm for additionally assessing the coverage of unique k-mers found in each species in a dataset.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `metagenomics_screening/profiling/krakenuniq/`
+  - `<sample_id>.classified.fastq.gz`: FASTQ file containing all reads that had a hit against a reference in the database for a given sample
+  - `<sample_id>.unclassified.fastq.gz`: FASTQ file containing all reads that did not have a hit in the database for a given sample
+  - `<sample_id>.report.txt`: A Kraken2-style report that summarises the fraction abundance, taxonomic ID, number of Kmers, taxonomic path of all the hits, with an additional column for k-mer coverage, that allows for more accurate distinguishing between false-positive/true-postitive hits
+  - `<sample_id>.classifiedreads.txt`: A list of read IDs and the hits each read had against each database for a given sample
+
+</details>
+
+The main taxonomic classification file from KrakenUniq is the `*report.txt` file. This is an extension of the Kraken2 report with the additional k-mer coverage information that provides more information about the accuracy of hits.
+
+You will only receive the `*.fastq.gz` and `*.classifiedreads.txt` file if you supply `--metagenomics_kraken_savereads` and/or `--metagenomics_kraken_save_readclassifications` parameters to the pipeline.
+
+:::info
+The output system of KrakenUniq can result in other `stdout` or `stderr` logging information being saved in the report file, therefore you must check your report files before downstream use!
+:::
+
+### Metagenomics Postprocessing
+
+#### taxpasta
+
+the output created by the `taxpasta merge` command. It combines the results of all the samples analyzed with a given metagenomic classifer by nf-core/eager. The file provides an overview of the classification results for all samples combined
```

Comment:
```suggestion
The output created by the `taxpasta merge` command. It combines the results of all the samples analyzed with a given metagenomic classifer by nf-core/eager in a standardised tabular taxon-table format. The file provides an overview of the classification results for all samples combined
```

Reply:
This should go below, the collapsible list, and the tool description should go here